src/components/orders-table/orders-data-table.component.tsx

```diff
@@ -45,6 +45,39 @@ interface OrdersDataTableProps {
   excludeCanceledAndDiscontinuedOrders?: boolean;
   useActivatedOnOrAfterDateFilter?: boolean;
 }
+interface CustomOrder extends Omit<Order, "orderer" | "patient"> {
+  patient: string;
+  patientUuid: string;
+  orderer: string;
+}
+
+function useSearchResults(tableEntries: Order[], searchString): CustomOrder[] {
+  const searchResults = useMemo(() => {
+    const flattenedData = tableEntries.map((eachObject) => {
+      return {
+        ...eachObject,
+        dateActivated: formatDate(parseDate(eachObject.dateActivated)),
+        patient: eachObject.patient?.display.split("-")[1],
+        patientUuid: eachObject.patient?.uuid,
+        status: eachObject.fulfillerStatus ?? "--",
+        orderer: eachObject.orderer?.display.split("-")[1],
+      };
+    });
+
+    if (searchString && searchString.trim() !== "") {
```

Comment:
You want to do this soft check before the mapping above so that we return early incases of an empty search term.